auth0/src/main/java/com/auth0/android/request/internal/BaseRequest.java

```diff
@@ -146,7 +148,7 @@ protected U parseUnsuccessfulResponse(Response response) {
     }
 
     @Override
-    public void onFailure(Request request, IOException e) {
+    public void onFailure(Call call, IOException e) {
```

Comment:
Why do we need the `Call` instance here? If we already know it failed, just pass `Request` as `call.request()`.